build.sbt

```diff
@@ -28,11 +28,11 @@ scalaVersion := "2.10.5"
 crossScalaVersions := Seq("2.10.5", "2.11.7")
 
 // Requires 0.8.2 or later and https://github.com/druid-io/druid/pull/1940
-val druid_version = "0.9.0-rc3-SNAPSHOT"
+val druid_version = "0.9.1-SNAPSHOT"
 // This is just used here for Path, so anything that doesn't break spark should be fine
 val hadoop_version = "2.4.0"
 // Requires a patch for https://issues.apache.org/jira/browse/SPARK-11016
-val spark_version = "1.5.2-mmx1"
+val spark_version = "1.5.2-mmx4"
```

Comment:
it is scoped `provided` so it shouldn't have anything in particular. The actual version is pulled during pull-deps